HW03/vitaly_kadyrko/vitaly_kadyrko_hw_03_t_02.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class Homework3
+  def task2(log)
+    output_array = []
+    good_data_array = log.split("\n").select { |x| x.length in 65..100 }
+    [] if good_data_array.size.zero?
+    good_data_array.each do |x|
+      x = x.split
```

Comment:
we usually don't overide existing variable, would be better to creat a new one 
```suggestion
      new_variable = x.split
```